aws-devicefarm-gradle-plugin/src/main/groovy/com.amazonaws.devicefarm/DeviceFarmServer.java

```diff
@@ -48,6 +50,20 @@
     private final DeviceFarmUploader uploader;
     private final DeviceFarmUtils utils;
 
+
+    /**
+     * Check whether the input test type is Appium.
+     *
+     * @param testType
+     * @return ture if it is an Appium test
+     */
+    private static boolean checkIfAppium(final TestType testType) {
+        if (testType.equals(TestType.APPIUM_JAVA_JUNIT) || testType.equals(TestType.APPIUM_JAVA_TESTNG) || testType.equals(TestType.APPIUM_PYTHON)) {
+            return true;
+        }
+        else return false;
```

Comment:
Since the `if` conditional is a `return`, you don't need the `else` here. I'd remove it and just `return false;`.